src/components/Common/Error.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { Field } from 'react-final-form'
+
+const defaultErrorStyles = {
+  color: 'red',
+  fontWeight: 'bold',
+  fontSize: '12px',
+  width: '100%',
+  height: '20px',
+}
+
+export const Error = ({ name, errorStyle }) => (
+  <Field
```

Comment:
I'm not sure I understand this. Is the `Field` component really necessary? Couldn't we just receive the whole `meta` object from props and use that?

If I'm understanding this correctly, this means that we'll have two `Field`s with the same `name` attribute, but one of them is used only to show errors. It doesn't sound right to me, is there a good reason for this approach?